matchers.go

```diff
@@ -57,6 +57,11 @@ func HTTP1Fast(extMethods ...string) Matcher {
 	return PrefixMatcher(append(defaultHTTPMethods, extMethods...)...)
 }
 
+// TLS matches HTTPS requests.
+func TLS() Matcher {
```

Comment:
Sure, here is a nice blogpost on the format: http://blog.fourthbit.com/2014/12/23/traffic-analysis-of-an-ssl-slash-tls-session